ipld/src/json.rs

```diff
@@ -1,6 +1,18 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
+//! Differences between serializers
+//!
+//! The serializer created here uses `multihash` and `libipld-json` uses plain `base64`.
+//! That means one has an extra `m` in front of all the encoded byte values, using our serializer.
+//!
+//! For example:
+//!
+//! |`forest_ipld`|`libipld-json`|
+//! | ----------- | ------------ |
+//! |```{ "/": { "bytes": "mVGhlIHF1aQ" } }```|```{ "/": { "bytes": "VGhlIHF1aQ" } }```|
+//!
+//! As `Lotus` is using `multihash-base64` and our serializer is using it as well.
```

Comment:
```suggestion
//! Since `Lotus` is also using `multihash-base64` and we're trying to be compatible, we cannot switch to `libipld-json`. It may be worthwhile to reconsider whether we want to stay compatible with Lotus in the future.
```